modules/storages/app/services/storages/group_folder_properties_sync_service.rb

```diff
@@ -260,13 +260,18 @@ def remote_group_users
         .call(storage: @storage, group: @storage.group)
     end
 
-    ### Model Scopes
-    def project_storage_scope
-      @storage.project_storages.automatic.joins(:project)
+    def format_and_log_error(error, context = {})
+      error_message = context.merge({ command: error.data.source,
+                                      message: error.log_message,
+                                      data: { status: error.data.payload.code, body: error.data.payload.body } })
+
+      OpenProject.logger.warn error_message.to_json
     end
 
+    ### Model Scopes
+
     def active_project_storages_scope
-      project_storage_scope.where(project: { active: true })
+      ::Storages::ProjectStorage.active_nextcloud_automatically_managed
```

Comment:
`project_storage_scope` here includes the `automatic` scope, is already scoped to a nextcloud storage (as only those are queried for the job) that has `automatically_managed` set to `true`, so unless you ran into some error (and i'd love to debug this with you) this change might not be needed.